website/project/model.py

```diff
@@ -2359,15 +2371,19 @@ def add_log(self, action, params, auth, foreign_user=None, log_date=None, save=T
             user=user,
             foreign_user=foreign_user,
             params=params,
+            node=self,
+            original_node=params['node']
         )
 
         if log_date:
             log.date = log_date
+        log.save()
 
-        self.date_modified = log.date.replace(tzinfo=None)
+        if len(self.logs) == 1:
```

Comment:
But I guess dates are rounded when they're stored in the db, so date_created and date_modified would be rounded to the second(above), self.logs[-1].date?

I feel like I wouldn't have done this if I hadn't noticed a discrepancy somewhere between the two when there shouldn't have been, but maybe it was just inside this method, there was still a discrepancy before it was saved, and then both were rounded.
![screen shot 2016-02-23 at 4 17 14 pm](https://cloud.githubusercontent.com/assets/9755598/13267035/09b65a9e-da49-11e5-908e-e58db01ed196.png)

![screen shot 2016-02-23 at 4 17 14 pm](https://cloud.githubusercontent.com/assets/9755598/13267017/e7f58e16-da48-11e5-8ff7-45a9e8652572.png)